src/features/basic/show-space-remaining.ts

```diff
@@ -0,0 +1,68 @@
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+import { storagesStore } from '@src/infrastructure/prun-api/data/storage';
+import { watchEffectWhileNodeAlive } from '@src/utils/watch';
+
+function formatFixed(f: number) {
```

Comment:
Instead of custom format functions, use ones from the `utils/format.ts`. See an example here: https://github.com/refined-prun/refined-prun/blob/5f4872a62ddaaf31466281f4960e26175002b156/src/features/basic/cxpo-auto-price.ts#L116